ui/package.json

```diff
@@ -34,6 +34,7 @@
         "azure-arm-resource": "^3.0.0-preview",
         "azure-arm-storage": "^3.1.0",
         "fs-extra": "^4.0.3",
+        "html-to-text": "^4.0.0",
```

Comment:
I feel this should be locked down, without the ^. There's quite some test code that essentially tests this package's functionality (rather than _how we use it_). A package update making our code fail (if the `parseIfHtml` behavior changes subtly) is more overhead, especially for this upstream repo which can break the downstream extensions.

Reply:
That's what the package-lock.json is for